composables/party/useParty.ts

```diff
@@ -57,23 +59,25 @@ export default <T>({ room, onMessage }: UsePartyParams<T>) => {
     wss.value.get(room.value)?.send(JSON.stringify(value))
   }
 
+  const closeConnection = () => {
+    for (const [room, ws] of wss.value) {
+      console.log(`[PARTY::CONNECTION] Closing connection with room ${room}`)
+      ws.close()
+      wss.value.delete(room)
+    }
+  }
+
   watch(
     room,
     (value) => {
-      if (value) {
+      if (value && preferencesStore.partyMode) {
```

Comment:
`useParty.ts` it's a generic composable , that might be used for other party types 

inside `useCursorParty` do something like 

![CleanShot 2024-04-08 at 10 24 58@2x](https://github.com/kodadot/nft-gallery/assets/44554284/8ab52808-fdb1-485a-90fa-aa54f4391b28)

and you can 
```suggestion
      if (value && !disabled) {
```